src/devices/video/ef9345.cpp

```diff
@@ -154,6 +171,7 @@ void ef9345_device::device_start()
 	save_item(NAME(m_latchm));
 	save_item(NAME(m_latchi));
 	save_item(NAME(m_latchu));
+	save_item(NAME(m_variant));
```

Comment:
If this is only set at construction and can't be changed, it shouldn't really be saved.  It can also be made `const` to prevent it from being changed after construction.  Sorry, didn't notice this before.